### ISSUE TITLE
Clarify parts of the coding guidelines

### ISSUE THREAD

**Issue by hsorby** [cellml/libcellml#315]
I would like to see the following added to the coding guidelines:

1. One statement of code per line due to code coverage requirements.
2. If statements always have braces that is no single statement ifs.

**Comment by agarny** [comment]
I am pretty sure that both of those are addressed by using Clang-Tidy and ClangFormat. To be confirmed once we have merged in PRs #288 and #310.

**Comment by hsorby** [comment]
Great.

**Comment by nickerso** [comment]
Neither #288 or #310 appear to have any documentation changes, so I'm guessing there should be some other changes to be made in the documentation...

**Comment by agarny** [comment]
I just meant that #288 and #310 would automatically fix code that fall in one of Hugh's two categories.

**Comment by nickerso** [comment]
yep, but those additional dependencies and how to use them should be covered in the documentation. And since it seems clang-format is going to be problematic on windows, it would still be useful, I think, to include @hsorby's proposed changes in the coding guidelines.

**Comment by agarny** [comment]
Oh, ok. Agreed.
